go.mod

```diff
@@ -24,3 +26,5 @@ require (
 	sigs.k8s.io/controller-runtime v0.3.0
 	sigs.k8s.io/yaml v1.1.0
 )
+
+replace k8s.io/client-go => k8s.io/client-go v0.0.0-20190918200256-06eb1244587a
```

Comment:
As I understand, sonobuoy has this dependency on a particular version of k8s/client-go and getting a newer version of client-go is breaking it. I hate to do this as well but i couldnt find a better way.

Reply:
If we can't fix this here, I propose we make `test/e2e` its own go module with the `replace` statement there.

Reply:
:grimacing: I think I would rather do the replace in the main `go.mod` file if we need to override it for sonobuoy, otherwise we risk shipping something different than we are testing.

Reply:
I take that back, as long as we can isolate the system under test, ensuring that the controller manager that is built is using the main `go.mod` file and only the test suite/utilities are using a different version of client-go, then this approach should be fine. It might also help to keep other testing dependencies from creeping into the main `go.mod` file as well.